xtask/src/commands/package/mod.rs

```diff
@@ -11,6 +13,16 @@ use structopt::StructOpt;
 use xtask::*;
 
 const INCLUDE: &[&str] = &["README.md", "LICENSE", "licenses.html"];
+pub(crate) const TARGET_MUSL_LINUX: &str = "x86_64-unknown-linux-musl";
+pub(crate) const TARGET_GNU_LINUX: &str = "x86_64-unknown-linux-gnu";
+pub(crate) const TARGET_WINDOWS: &str = "x86_64-pc-windows-msvc";
+pub(crate) const TARGET_MACOS: &str = "x86_64-apple-darwin";
```

Comment:
Is it worth naming `TARGET_MACOS` `TARGET_X86_64_MACOS` to allow for the eventual arrival of `aarch64-apple-darwin` as `TARGET_ARM_64_MACOS`?